the-collab-lab/tcl-30-whats-near-me
# [TCL30-5] Render pins on the map from "nearby" API

## Description
This PR adds a pin component to the app and render pins on the map using data fetched from "nearby" API

## Related Issue
Closes [TCL30-9](https://the-collab-lab.atlassian.net/browse/TCL30-9)

## Acceptance Criteria
- Data fetched from "nearby" API is used to render pins on the map
- Pins should be 24px x 24px in size and should have a little down caret on the bottom edge of the pin pointing to the location of the place
- Pins should show a scaled-down thumbnail image in cases when a thumbnail image is present
- Pins should render a default pin image when no thumbnail is returned
- Pins should flow in onto the map

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before

![before_upd](https://user-images.githubusercontent.com/16920088/126311118-b0ea9465-b3ff-424f-b15c-a7bc0f97532f.png)


### After

![after_upd](https://user-images.githubusercontent.com/16920088/126310917-8822e8b6-214e-4eb1-bb6d-2acdd8580f4a.png)


## Testing Steps / QA Criteria
- From your terminal, pull down this branch
-  Run ``npm start`` to launch the app
-  Make sure pins are rendered on the map
-  Make sure pins show a scaled-down thumbnail image in cases when a thumbnail image is present and their size is 24x24
-  Make sure pins render a default pin image when no thumbnail is returned
- Make sure pins flow in onto the map
